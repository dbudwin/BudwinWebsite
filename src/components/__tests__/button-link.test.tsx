import "@testing-library/jest-dom";

import ButtonLink from "../button-link";
import React from "react";
import renderer from "react-test-renderer";

it("creates a button link that will open a new tab securely", () => {
    const buttonLink = renderer.create(<ButtonLink url={"http://www.budw.in/"} text={"My Website"} icon={"coffee"} />).toJSON();

    expect(buttonLink).toMatchSnapshot();
});
