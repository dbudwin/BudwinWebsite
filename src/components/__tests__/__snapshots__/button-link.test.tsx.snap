// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a button link that will open a new tab securely 1`] = `
<a
  className="button is-link is-info"
  href="http://www.budw.in/"
  rel="noopener noreferrer"
  target="_blank"
>
  <span
    className="icon is-small"
  >
    <i
      className="fa"
    />
  </span>
  <span>
    My Website
  </span>
</a>
`;
